libs/perun/components/src/lib/services-status-list/services-status-list.component.ts

```diff
@@ -124,6 +150,12 @@ export class ServicesStatusListComponent implements OnChanges, AfterViewInit {
             return item[property];
         }
       };
+      this.dataSource.filterPredicate = (data: ServiceState, filter: string) => {
+        return customDataSourceFilterPredicate(data, filter, this.displayedColumns, this.getDataForColumn, this)
+      };
+      this.dataSource.sortData = (data: ServiceState[], sort: MatSort) => {
+        return customDataSourceSort(data, sort, this.getDataForColumn, this);
+      };
```

Comment:
when sorting by ID - its not working - no error appears but it doesnt do anything
when sorting by end and start date the error in console aapears
![image](https://user-images.githubusercontent.com/32871167/107033579-ce21df80-67b5-11eb-80a7-a4e3c5d8680e.png)